addons/point_of_sale/static/src/app/models/utils/compute_combo_lines.js

```diff
@@ -23,7 +23,7 @@ export const computeComboLines = (
             decimalPrecision.find((dp) => dp.name === "Product Price").digits
         );
         remainingTotal -= priceUnit;
-        if (comboLine == childLineConf[childLineConf.length - 1]) {
+        if (comboLine.id == childLineConf[childLineConf.length - 1].combo_line_id.id) {
```

Comment:
I forgot about that, but there isn't a possibility that `childLineConf[childLineConf.length - 1]` or `childLineConf[childLineConf.length - 1].combo_line_id` is undefined ?

Reply:
I suppose no because we iterate through `childLineConf` with `for (const conf of childLineConf)` so we will reach `childLineConf[childLineConf.length - 1]` and we have `comboLine = conf.combo_line_id;` meaning `childLineConf[childLineConf.length - 1]` is defined and we have `combo = comboLine.combo_id;` meaning `childLineConf[childLineConf.length - 1].combo_line_id` is defined
Conclusion we would already have the error anyway

Reply:
Just rebase, and I will r+